src/Pyz/Yves/CheckoutPage/Theme/default/views/address/address.twig

```diff
@@ -24,10 +25,36 @@
             isMultipleShipmentEnabled: data.isMultipleShipmentEnabled,
             jsAddressClass: jsAddressClass,
             isBillingSameAsShippingChecked: billingSameAsShippingData is not defined or billingSameAsShippingData,
+            formSubmitClassName: formSubmitClassName,
         },
     } only %}
         {% block fields %}
-            {% set hasShippingAddresses = data.form.shippingAddress.id_customer_address is defined %}
+            {% set deliveryContainerClassName = embed.jsAddressClass ~ '__delivery' %}
+            {% set singleDeliveryContainerClassName = embed.jsAddressClass ~ '__single-delivery' %}
+            {% set deliverySelectClassName = embed.jsAddressClass ~ '__form-select-shipping-address' %}
+            {% set validatorTriggerClassName = embed.jsAddressClass ~ '__validator-trigger' %}
+            {% set shippingValidationContainerClassName = embed.jsAddressClass ~ '__shipping-validation-container' %}
+            {% set shippingClassName = embed.jsAddressClass ~ '__shipping' %}
+            {% set multishipmentTargetClassName = embed.jsAddressClass ~ '__items-target' %}
+            {% set validatorClassName = embed.jsAddressClass ~ '__validate-next-checkout-step' %}
+            {% set multishipmentValidatorClassName = embed.jsAddressClass ~ '__multishipment-validate-next-checkout-step' %}
+            {% set billingSameAsShippingContainerClassName = embed.jsAddressClass ~ '__billing-same-as-shipping-container' %}
+            {% set billingSameAsShippingClassName = embed.jsAddressClass ~ '__billing-same-as-shipping' %}
+            {% set selectBillingAddressClassName = embed.jsAddressClass ~ '__form-select-billing-address' %}
+            {% set billingClassName = embed.jsAddressClass ~ '__billing' %}
+            {% set billingFormClassName = embed.jsAddressClass ~ '__billing-form' %}
+            {% set hiddenClassName = 'is-hidden' %}
+
+            {% set shippingAddress = data.form.shippingAddress.id_customer_address %}
+            {% set shippingAddressValue = shippingAddress.vars.value %}
```

Comment:
The `shippingAddress` can be undefined and it leads to an error in the `shippingAddressValue`.
Please check the workflow with a customer without saved addresses.